Gemfile

```diff
@@ -28,6 +28,8 @@ gem "jbuilder"
 
 gem "rspec"
 
+gem "rubocop", ">= 1.66.1"
```

Comment:
We only need Rubocop in the development environment. We don't need it in other environments. Fix it